frontend-angularjs/app/views/schedules.html

```diff
@@ -126,19 +126,12 @@ <h2 class="text-center" style="padding-bottom: 35px" ng-show="page.schedules.len
                             </div>
                         </div>
                         <div class="form-group">
-                            <label for="inputAvailability" class="col-md-2 control-label">Опис</label>
+                            <label for="inputAvailability" class="col-md-2 control-label">Доступнiсть</label>
 
-                            <div class="col-md-10">
-                                <div class="checkbox" id="inputAvailability">
-                                    <button ng-click="selectDay('MONDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Пн</button>
-                                    <button ng-click="selectDay('TUESDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Вт</button>
-                                    <button ng-click="selectDay('WEDNESDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Ср</button>
-                                    <button ng-click="selectDay('THURSDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Чт</button>
-                                    <button ng-click="selectDay('FRIDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Пт</button>
-                                    <button ng-click="selectDay('SATURDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Сб</button>
-                                    <button ng-click="selectDay('SUNDAY')" style="padding:10px" type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false">Вс</button>
-                                </div>
-g
+                            <div class="col-md-10" id="inputAvailability">
```

Comment:
Those checkboxes will never change. Hardcode them with correct values without blackmagic how it was in the mockup.
```
<div class="form-group">
                            <label for="inputAvailability" class="col-md-2 control-label">Опис</label>

                            <div class="col-md-10">
                                <div class="checkbox" id="inputAvailability" style="padding-top: 16px;">
                                    <label>
                                        <input type="checkbox" checked="checked"> Пн
                                    </label>
                                    <label>
                                        <input type="checkbox" checked="checked"> Вт
                                    </label>
                                    <label>
                                        <input type="checkbox" checked="checked"> Ср
                                    </label>
                                    <label>
                                        <input type="checkbox" checked="checked"> Чт
                                    </label>
                                    <label>
                                        <input type="checkbox" checked="checked"> Пт
                                    </label>
                                    <label>
                                        <input type="checkbox"> Сб
                                    </label>
                                    <label>
                                        <input type="checkbox"> Нд
                                    </label>

                                </div>
                            </div>
                        </div>
```